cloud/disk_manager/internal/pkg/clients/nbs/interface.go

```diff
@@ -98,6 +98,14 @@ const (
 	CheckpointTypeWithoutData
 )
 
+type CheckpointStatus uint32
+
+const ( // Must be in same order as protos.ECheckpointStatus.
```

Comment:
Alternatively we might have function mapping between these two enums via switch case